docs/builds/guides/integration/configuration.md

```diff
@@ -133,6 +133,50 @@ ClassicEditor
 	} );
 ```
 
+### Extended format
+
+You can use the extended toolbar configuration format to access additional options:
+
+```js
+    toolbar: {
+        items: [ 'bold', 'italic', '|', 'undo', 'redo', '-', 'numberedList', 'bulletedList' ],
+        viewportTopOffset: 30,
+        shouldNotGroupWhenFull: true
+    }
+```
+
+ * **`items`** &ndash; An array of toolbar item names. The components (buttons, dropdowns, etc.) which can be used as toolbar items are defined in `editor.ui.componentFactory` and can be listed using the following snippet: `Array.from( editor.ui.componentFactory.names() )`
+
+ * **`viewportTopOffset`** &ndash; The offset (in pixels) from the top of the viewport used when positioning a sticky toolbar.
+ Useful when a page with which the editor is being integrated has some other sticky or fixed elements
+ (e.g. the top menu). Thanks to setting the toolbar offset the toolbar will not be positioned underneath or above the page's UI.
+
+ * **`shouldNotGroupWhenFull`** &ndash; When set `true`, the toolbar will stop grouping items and let them wrap to the next line when there is not enough space to display them in a single row.
```

Comment:
We should mention here that grouping is enabled by default.

Reply:
Ok. By the way, I am considering if it will be more clear if the name will be `groupWhenFull`? (ofc. enabled by default).